packages/and/tests/andModule.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon'
+import {AndModule} from '../src'
+
+import {Validator, ConstraintExecuter} from '@configurable-validator/core'
+
+describe('AndModule', () => {
+    
+    let validatorMock: Validator;
+
+    beforeEach(() => {
+        validatorMock = sinon.mock(Validator)
```

Comment:
probably you would want `sinon.createStubbedInstance`, but doesn't really matter because ithink don't work yet